src/Notifier.php

```diff
@@ -105,9 +111,13 @@ public function buildNotice($exc)
             ],
             'os' => php_uname(),
             'language' => 'php '.phpversion(),
+            'rootDirectory' => $this->opt['rootDirectory'] ?: $_SERVER['DOCUMENT_ROOT'] ?: null,
```

Comment:
Can you please remove double `?:` usage? It is not obvious what is the result of this statement...